src/basics.php

```diff
@@ -142,7 +142,13 @@ function dd($var, $showHtml = null): void
             'line' => $trace[0]['line'],
             'file' => $trace[0]['file'],
         ];
-
+        
+        $origin = substr($_SERVER['HTTP_REFERER'], 0, (strlen($_SERVER['HTTP_REFERER']) - 1));
```

Comment:
I think this would be `env('HTTP_REFERER')`

Reply:
maybe all moments you need to debug, you are on dev env, and not need verify referer.

Reply:
`$_SERVER['HTTP_REFERER']` could not be set and would trigger a notice, right?